provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/publish.yml

```
@@ -73,7 +73,7 @@ jobs:
         {
           echo 'summary<<EOF'
           if [[ "$LAST_VERSION" != "No stable release" ]]; then
-            schema-tools compare --provider="#{{ .Config.provider }}#" --old-commit="$LAST_VERSION" --new-commit="--local-path=provider/cmd/pulumi-resource-#{{ .Config.provider }}#/schema.json"
+            schema-tools compare --provider="#{{ .Config.provider }}#" --old-commit="$LAST_VERSION"#{{ if ne .Config.organization "pulumi" }}# --repository="github://api.github.com/#{{ .Config.organization }}#"#{{ end }}# --new-commit="--local-path=provider/cmd/pulumi-resource-#{{ .Config.provider }}#/schema.json"
```

What about always using the config organization - one less conditional where only 1 branch might actually ever be tested.

```suggestion
            schema-tools compare --provider="#{{ .Config.provider }}#" --old-commit="$LAST_VERSION" --repository="github://api.github.com/#{{ .Config.organization }}#" --new-commit="--local-path=provider/cmd/pulumi-resource-#{{ .Config.provider }}#/schema.json"
```

---

@danielrbradley That's what I had in the beginning, but I did the extra work to prevent unnecessary changes to the Pulumi managed providers. I will get rid of the `if` you like that better.